omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientsearch/PatientSearchWidgetFragmentController.java

```diff
@@ -34,10 +34,18 @@ public void controller(FragmentModel model, UiSessionContext sessionContext,
                            @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
                            @FragmentParam(value = "showLastViewedPatients", required = false) Boolean showLastViewedPatients,
                            @FragmentParam(value = "initialSearchFromParameter", required = false) String searchByParam,
-                           @FragmentParam(value = "registrationAppLink", required=false) String registrationAppLink) {
+                           @FragmentParam(value = "registrationAppLink", required=false) String registrationAppLink,
+                           @FragmentParam(value = "searchOnExactIdentifier", required=false) Boolean searchOnExactIdentifier) {
```

Comment:
Does this actually do anything?  It's not needed, I don't think.... but it could be nice to keep this and if it's !=NULL then set this value as the "searchOnExactIdentifier" model attribute instead of determining based on OpenMRS version.

Reply:
Hey Mr. Goodrich!
I was looking at this comment and I was trying to understand the logic that you were proposing here. So searchOnExactIdentifier should be true or not-null in the case that the OpenMRS version is at least 2.1, so I guess I don't understand how to not change it or set it to not null unless I am still determining version?

Thanks for any clarity! 
Joe

Reply:
As the code is written now, this line doesn't do anything, and could be removed, correct?  The parameter "Boolean searchOnExactIdentifier" is never referenced later on in the method, is it?

Reply:
Yes that’s correct. So I’m all for removing that. I’m not sure why I had thought it was needed. But I was referring to your comment about leaving it in and using it to set the value instead of determining the version?